addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -15,6 +15,269 @@ $MAX-Z-INDEX : 2147483647 !default;
     object-position: top;
 }
 
+
+// Common to new slides pages
+// **************************************************
+
+.o_wslides_wrap {
+    .text_small_caps {
+        font-variant: small-caps;
+    }
+    .image-muted {
+        opacity: 0.5;
+    }
+    .cta-title {
+        font-size: 1.2em;
+    }
+}
+
+
+// New home page
+// **************************************************
+
+div .o_wslides_home_nav {
+    position: relative;
+    top: -50px;
+}
+
+div .o_wslides_home_main {
+    .o_wslides_home_header {
+    }
+    .o_wslides_home_footer {
+    }
+
+    .o_wslides_home_aside {
+        .fa-check {
+            color: #00A09D;
+            font-size: 1.2em;
+        }
+    }
+    .o_wslides_home_content_section_title, .o_wslides_home_aside_title {
+        h3 {
+            font-size: 1.4em;
+        }
+        h4 {
+            font-size: 1.3em;
+        }
+    }
+
+    .o_wslides_course_card {
+        .card-body {
+            padding: 0.75em 1em;
+        }
+        .card-footer {
+            padding-top: 0.75em 1em;
+        }
+    }
+    // responsive card-deck like
+    .row.o_wslides_row > div[class*='col-'] {
+        display: flex;
+        flex:1 0 auto;
+        .card {
+            width:100%;
+        }
+    }
+
+    // tools
+    // ****************************************
+    .text_small_caps {
+        font-variant: small-caps;
+    }
+
+    .image-muted {
+        opacity: 0.5;
+    }
+
+    .o_wslides_home_circled {
+        border-radius: 50%;
+        width: 1.2em;
+        height: 1.2em;
+        padding: 0.25em 0.5em 0.25em 0.5em;
+        border: 0.2em solid #666;
+        text-align: center;
+        box-sizing: initial;
+    }
+
+    .progress {
+        overflow: visible;
+        margin-bottom: 0em;
+        height: 4px;
+        .progress-bar {
+            background-color: #00A09D;
+            position: relative;
+            border-radius: 4px;
+            span {
+                background-color: #00A09D;
+                position: absolute;
+                bottom: -20px;
+                font-size: 10px;
```

Comment:
osef right now but I think the standard is now to put sizes in "rem".
So you know for next time.

Reply:
Ok, will be done in a cleaning pass soon. Thanks for noting, I tried to remove those but some are still let :) . I put it into a post-it.